src/triangle_application.cpp

```diff
@@ -218,6 +218,9 @@ std::vector<const char*> TriangleApplication::GetRequiredExtensions() {
         extensions.push_back(VK_EXT_DEBUG_UTILS_EXTENSION_NAME);
     }
 
+    extensions.push_back(
```

Comment:
Add a comment explaining that VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME is the instance extension.